suites/reef/cephfs/tier-0_cephfs_upgrade_ibm_6x_to_7x.yaml

```diff
@@ -0,0 +1,201 @@
+#===============================================================================================
+# Cluster Configuration:
+#    conf/quincy/cephfs/tier-2_cephfs_upgrade.yaml
+#
+# Test Steps:
+#   (1) Bootstrap cluster using latest released ceph 5.x with below options,
+#       - rhcs-version: 5.2 in this suite, but it can be changed to 5.x for upgrade start version as needed
```

Comment:
Please fix the 5.x, 5.2, rhcs-version statements and add comments to match the ibm version 6.x upgrade suite

Reply:
done